common/rdm/AckTimerResponder.cpp

```diff
@@ -220,6 +220,7 @@ const RDMResponse *AckTimerResponder::ResponseFromQueuedMessage(
           queued_response->ParamDataSize());
       break;
     case RDMCommand::SET_COMMAND_RESPONSE:
+      // coverity(SWAPPED_ARGUMENTS)
       return new RDMSetResponse(
```

Comment:
I was looking at this the other day. Shouldn't we create an RDMSetResponse constructor that takes a SetRequest and does half the leg work for you?